app/src/main/java/com/example/periodtracker/Health.java

```diff
@@ -0,0 +1,43 @@
+package com.example.periodtracker;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+public class Health extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_health);
+        // add this to the bottom of pages for navigation
+        // Initialize and assign variable
+        BottomNavigationView bottomNavigationView=findViewById(R.id.bottom_navigation);
+
+        // Set to highlight current page
+        bottomNavigationView.setSelectedItemId(R.id.health);
+
+        // Perform item selected listener
+        bottomNavigationView.setOnItemSelectedListener(item -> {
```

Comment:
Can you explain how this works? I cannot seem to find how you setup bottom navigation, if this can be made more maintainable, similar to the onClickListener that would be best but if not, don't worry.

Reply:
bottomNavigationView.setSelectedItemId(R.id.health); will highlight the navigation icon, for this case, the health icon, to represent which page we are on.

bottomNavigationView.setOnItemSelectedListener(item -> { will turn on the listener for the navigation bar and perform actions when there are changes to it essentially